src/PrestaShopBundle/Form/Admin/Improve/Design/Pages/CmsPageType.php

```diff
@@ -150,6 +172,12 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 ],
             ])
             ->add('meta_keyword', TranslatableType::class, [
+                'label' => $this->trans('Meta keywords', 'Admin.Global'),
+                'help' => sprintf(
+                    '%s %s',
```

Comment:
This looks like a wording?

Reply:
What do you mean? I've concatenated 2 different strings using `sprintf` which is more readable imo.

Reply:
Okay seems plausible